neb/inc/com/centreon/engine/nebstructs.hh

```diff
@@ -320,7 +320,8 @@ typedef struct nebstruct_flapping_struct {
   struct timeval timestamp;
 
   int flapping_type;
-  char* host_name;
+  uint64_t host_id;
+  uint64_t service_id;
   char* service_description;
```

Comment:
Why keeping service_description? Is it really needed if you have the service_id?